src/apps/admin/app.py

```
@@ -101,6 +104,48 @@ async def get_repository_status():
+        media_type="text/event-stream",
+        headers={
+            "Cache-Control": "no-cache",
+            "Connection": "keep-alive",
+            "Access-Control-Allow-Origin": "*",
+        },
+    )
+
+
+@app.post("/api/build-index")
+async def build_index_stream(
+    coordinator: SyncCoordinator = Depends(get_sync_coordinator),
```

Ruff (0.12.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)